### ISSUE TITLE
 is this method suitable for multi-dimensional data

### ISSUE THREAD

**Issue by 13121283123** [jjdabr/forecastNet#11]
Excuse me, is this method suitable for multi-dimensional data? if so, how is it implemented? I encounter the following problems in the process of implementation. 
Next_cell_input = torch.cat ((input, hidden, target [i,:,:]), dim=1). 

Suppose the dimension of input is (16，24，4) (batch-size,input-length,diemnsion) the dimension of hidden can be set to （16，24，1）, but the dimension of the final targets [i，：，；] is （16，1）, so how can them be spliced together? because the predicted value is still one-dimensional, but the input is multi-dimensional, This makes it impossible for them to splice, like this input:(16,24,4)hidden:(16,24,1) target[i,:,:] (16,1)how do you achieve the splice, thank you very much

**Comment by jjdabr** [comment]
Thank you for your interest. In its current form, it is intended as a univariate model. A multivariate form will be released soon. However, in the mean time, you can easily adapt the current model for multivariate data. For this, simply collapse all dimensions onto the "input_seq_length" dimension of your input. For example, if your input is [input_seq_length, batch_size, input_dim] = [16, 24, 4], reshape this to a [16*4, 24, 1] = [64, 24, 1] tensor.

**Comment by 13121283123** [comment]
Thank you very much&nbsp; for your reply



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Joel Janek ***@***.***&gt;; 
发送时间: 2021年5月22日(星期六) 凌晨4:47
收件人: ***@***.***&gt;; 
抄送: "Changxia ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [jjdabr/forecastNet]  is this method suitable for multi-dimensional data (#11)





 
Thank you for your interest. In its current form, it is intended as a univariate model. A multivariate form will be released soon. However, in the mean time, you can easily adapt the current model for multivariate data. For this, simply collapse all dimensions onto the "input_seq_length" dimension of your input. For example, if your input is [input_seq_length, batch_size, input_dim] = [16, 24, 4], reshape this to a [16*4, 24, 1] = [64, 24, 1] tensor.
